health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -481,6 +481,47 @@ const createUpdatePlanProject = async (req) => {
+        });
+        // const currentCategory = 
+        const prevFormulaValues =
+          fetchedPlanForSubFormula?.operations.length > 0
+            ? fetchedPlanForSubFormula?.operations?.map((row) => {
+                const updatedRow = {
+                  ...row,
+                  active: false,
+                };
+                return updatedRow;
+              })
+            : [];
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)